model/relation/event/MediaSavedEvent.php

```diff
@@ -26,6 +26,28 @@
 
 class MediaSavedEvent implements Event
 {
+    /** @var string */
+    private $mediaId;
+
+    /** @var array */
+    private $elementIds;
```

Comment:
```suggestion
    private $mediaIds;
```

Since it cannot be different than other mediaIds, right?

Reply:
Im going to put `mediaReferenes` to distinguish from `media`, wdyt?